ai/ollama/v0/main.go

```diff
@@ -0,0 +1,100 @@
+//go:generate compogen readme ./config ./README.mdx
+package ollama
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"sync"
+
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/instill-ai/component/base"
+)
+
+const (
+	TaskTextGenerationChat = "TASK_TEXT_GENERATION_CHAT"
+	TaskTextEmbeddings     = "TASK_TEXT_EMBEDDINGS"
+	EndPoint               = "endpoint"
```

Comment:
question
`	EndPoint               = "endpoint"`
What is it for?

Reply:
removed this variable